x/confidentialtransfers/client/cli/query.go

```diff
@@ -178,3 +186,126 @@ func queryAllAccounts(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+// GetCmdQueryTx implements a command to query a tx by it's transaction hash and return it in it's decrypted state.
+func GetCmdQueryTx() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tx [hash]",
+		Short: "Query the confidential transaction and decrypts it",
+		Long: "Query the confidential transaction by it's tx hash and decrypts it using the private key of the account in --from" +
+			"Pass the --from flag to decrypt the account" +
```

Comment:
Maybe let's rename this flag to --address instead or similar?

Reply:
Renamed to --decryptor here